cmake/FindUnibilium.cmake

```diff
@@ -23,6 +23,10 @@ find_path(UNIBILIUM_INCLUDE_DIR unibilium.h
           PATHS ${PC_UNIBILIUM_INCLUDEDIR} ${PC_UNIBILIUM_INCLUDE_DIRS}
           ${LIMIT_SEARCH})
 
+if(WIN32)
+  set(UNIBILIUM_USE_STATIC ON)
```

Comment:
Why force it always? Instead, perhaps a new option like `NVIM_BUILD_STATIC`. Currently it will only work for Windows, but we may later add support for static builds on unix.

Reply:
Uniblium in mingw64 only static libs. To declare use static lib for disambiguation

Reply:
All of this static stuff should be controlled by a single global option, `NVIM_BUILD_STATIC`. 

Instead of checking `if(WIN32)`, check `if(NVIM_BUILD_STATIC)`.

Reply:
Like NVIM_BUILD_STATIC, give choice right to dev/users. I'll add that.